Lib/sysconfig.py

```diff
@@ -569,6 +576,8 @@ def get_config_vars(*args):
         except AttributeError:
             # sys.abiflags may not be defined on all platforms.
             _CONFIG_VARS['abiflags'] = ''
+        _CONFIG_VARS['implementation'] = _get_implementation()
+        _CONFIG_VARS['implementation_lower'] = _get_implementation().lower()
```

Comment:
Not sure if this is the correct name; we do have `sys.implementation` with names like `cpython`, `jython`, etc.  The usage here seems different, more like a directory name to find libraries.

Reply:
What would be a better name? We need something that maps like  CPython -> `python`, PyPy -> `pypy`. I don't know what Pyston/Jython/Graal python use, maybe some of them want to reuse the `python` name?

Reply:
My point is that `cpython` is an implementation name, `Python` isn’t.

Better name: any idea if we don’t look at the values but the usage?  Isn’t it the name for a directory containing libraries or modules?

Reply:
I agree, ``implementation`` does not fit well. There is ``platlibdir``. How about something like ``impllibdir`` for "implementation library directory"?

Reply:
Adopted `impllibdir`